pkg/security/tests/chown32_test.go

```diff
@@ -323,7 +318,7 @@ func checkSyscallTester(t *testing.T, path string) {
 	t.Helper()
 	sideTester := exec.Command(path, "check")
 	if _, err := sideTester.CombinedOutput(); err != nil {
-		t.Skip()
+		t.Skip("cannot run syscall tester check")
```

Comment:
When is this supposed to happen ? Isn't it an error ?

Reply:
The idea behind this is just to check that the ia32 executable is able to run, until now if it was not able to run (in some docker cases for example) it was not an error but just a skip. On the other hand I think you're right and we should enforce this coverage starting from now. I will fix this